pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -136,27 +189,46 @@ void Panels::GSWindowSettingsPanel::AppStatusEvent_OnSettingsApplied()
 
 void Panels::GSWindowSettingsPanel::ApplyConfigToGui( AppConfig& configToApply, int flags )
 {
-	const AppConfig::GSWindowOptions& conf( configToApply.GSWindow );
+	const AppConfig::GSWindowOptions &conf(configToApply.GSWindow);
+
+	if (!(flags & AppConfig::APPLY_FLAG_FROM_PRESET)) { // Presets don't control these values
+		m_check_CloseGS					->SetValue(conf.CloseOnEsc);
+		m_check_Fullscreen				->SetValue(conf.DefaultToFullscreen);
+		m_check_HideMouse				->SetValue(conf.AlwaysHideMouse);
+		m_check_SizeLock				->SetValue(conf.DisableResizeBorders);
+
+		m_combo_AspectRatio				->SetSelection(enum_cast(conf.AspectRatio));
+
+		m_combo_ScalingType				->SetSelection(enum_cast(conf.ScalingType));
+		m_combo_ScalingType				->Enable(conf.AspectRatio != AspectRatio_Stretch);
 
-	if( !(flags & AppConfig::APPLY_FLAG_FROM_PRESET) )	
-	{//Presets don't control these values
-		m_check_CloseGS		->SetValue( conf.CloseOnEsc );
-		m_check_Fullscreen	->SetValue( conf.DefaultToFullscreen );
-		m_check_HideMouse	->SetValue( conf.AlwaysHideMouse );
-		m_check_SizeLock	->SetValue( conf.DisableResizeBorders );
+		m_combo_FMVAspectRatioSwitch	->SetSelection(enum_cast(conf.FMVAspectRatioSwitch));
+		m_combo_FMVAspectRatioSwitch	->Enable(conf.AspectRatio != AspectRatio_4_3);
 
-		m_combo_AspectRatio	->SetSelection( (int)conf.AspectRatio );
-		m_text_Zoom			->ChangeValue( conf.Zoom.ToString() );
+		m_text_Zoom						->ChangeValue(conf.Zoom.ToString());
 
-		m_check_DclickFullscreen ->SetValue( conf.IsToggleFullscreenOnDoubleClick );
-		m_check_AspectRatioSwitch->SetValue( conf.IsToggleAspectRatioSwitch );
+		m_check_ScalingCompensation		->SetValue(conf.DisableScalingCompensation);
+		m_check_ScalingCompensation		->Enable((conf.AspectRatio != AspectRatio_Stretch &&
+			conf.ScalingType != ScalingType_Fit) || conf.AspectRatio == AspectRatio_Frame);
 
-		m_text_WindowWidth	->ChangeValue( wxsFormat( L"%d", conf.WindowSize.GetWidth() ) );
-		m_text_WindowHeight	->ChangeValue( wxsFormat( L"%d", conf.WindowSize.GetHeight() ) );
+		m_check_DclickFullscreen 		->SetValue(conf.IsToggleFullscreenOnDoubleClick);
+
+		m_text_WindowWidth				->ChangeValue(wxsFormat(L"%d", conf.WindowSize.GetWidth()));
+		m_text_WindowHeight				->ChangeValue(wxsFormat(L"%d", conf.WindowSize.GetHeight()));
```

Comment:
IMO all that whitespace in between hurts readability.

Reply:
The entire file could actually use an update. The code is a complete mess.

Reply:
> The entire file could actually use an update. The code is a complete mess.

Pretty much all files in the PCSX2 core are a mess. ;)

Reply:
Ye but this is special, messy on a new level xD

Reply:
Not really. This is quite average. What we really need is to clang-format the entire emulator code.

Reply:
Are there any side effects doing that ?

Reply:
It would require an extra click when looking at the code blame history.
And it would require all future commiters to clang-format (pretty much) all the files they change, or the automatic (travis) build will fail.